NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -277,6 +284,10 @@ static void ReadNicConfiguration(PARANDIS_ADAPTER *pContext, PUCHAR pNewMACAddre
             GetConfigurationEntry(cfg, &pConfiguration->USOv6Supported);
 #endif
 
```

Comment:
style: the rest of this area is back-to-back for if/endif statements, not sure if we want these new lines